pages/linux/ceph.md

```diff
@@ -0,0 +1,32 @@
+# ceph
+
+> Ceph is a free-software storage platform, implements object storage on a single distributed computer cluster, and provides interfaces for object, block and file-level storage.
```

Comment:
We tend to omit adjectives ("free-software", "open source", "top quality") in the main description. Stick to describing what it is and what it does.